spec/infinispan_local_spec.js

```diff
@@ -81,6 +81,24 @@ describe('Infinispan local client', function() {
     .catch(t.failed(done))
     .finally(done);
   });
+  it('can connect to server with lower protocol version', function(done) {
```

Comment:
This test is not needed. The CI runs the entire testsuite (not only a subset of operations) with protocols 2.2 and 2.5. See the [Jenkinsfile](https://github.com/infinispan/js-client/blob/master/Jenkinsfile).

Also, one note about the code. If the array would have more elements, once the first execution completed, the test would have finished, skipping the other runs.

Reply:
Btw, the testsuite is run with 3 different protocol versions: `2.8`, `2.5` and `2.2`. If you really wanted to try out a lower protocol it should run with both `2.5` and `2.2`. The way the test is designed you'd only have run with `2.2`.

Reply:
Hmmm, the test is still there?

Reply:
Fixed.